plugins/poll/assets/javascripts/discourse/components/poll-option-irv-dropdown.gjs

```diff
@@ -0,0 +1,58 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { fn } from "@ember/helper";
+import { action } from "@ember/object";
+import DButton from "discourse/components/d-button";
+import DropdownMenu from "discourse/components/dropdown-menu";
+import icon from "discourse-common/helpers/d-icon";
+import I18n from "discourse-i18n";
+import DMenu from "float-kit/components/d-menu";
+
+export default class PollOptionsDropdownComponent extends Component {
+  @tracked rank;
+
+  constructor() {
+    super(...arguments);
+    this.rank = this.args.rank;
```

Comment:
why don't we use `this.args.rank` directly? That would remove the need for this and `@tracked rank`

Reply:
@jjaffeux because `rank` is an attribute of options and it doesn't currently re-render on a change to this sub-argument.  If I remove the local variable and just rely on the incoming value of `this.args.rank` the interface is unresponsive to the change.  If you can propose a way for that to work, I'm all ears.